compass/landice/tests/calving_dt_convergence/namelist.landice

```diff
@@ -0,0 +1,30 @@
+config_start_time = '0000-01-01_00:00:00'
+config_run_duration = '0025-00-00_00:00:00'
+config_adaptive_timestep = .true.
+config_adaptive_timestep_CFL_fraction = 0.95
```

Comment:
Why this value instead of the usual 0.8?

Reply:
To be as aggressive as possible with preventing advection from limiting the CFL.